compiler/rustc_borrowck/src/borrowck_errors.rs

```diff
@@ -351,53 +315,36 @@ impl<'cx, 'tcx> crate::MirBorrowckCtxt<'cx, 'tcx> {
         immutable_section: &str,
         action: &str,
```

Comment:
I think that this will need more changes because `action` is likely containing English words that would need to be in the Fluent message to be translated.